lib/modules/platform/github/index.ts

```diff
@@ -1909,5 +1912,5 @@ export async function commitFiles(
   // and return the remote commit SHA
   await git.resetToCommit(commitResult.parentCommitSha);
   const commitSha = await git.fetchBranch(branchName);
-  return commitSha;
+  return commitSha as LongCommitSha;
```

Comment:
I think it would be better if git.fetchBranch returned LongCommitSha instead of using `as` here?

Reply:
it does, going to remove this cast as it's not needed :)